<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapTematikConfig</class>
 <widget class="QDialog" name="MapTematikConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Borders Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Transparency</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Borders Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Borders Style</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Label with</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdvance">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Advance...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBorderStyle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="sliderTransparency">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxPickColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Pick a Color Scheme</string>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLabel" name="labelNumberClass">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Data Class</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Label Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="textColorBorder">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textColorBackground">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textColorLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBorderColor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBackgroundColor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLabelColor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>520</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCentroid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Centroid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse Over</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMoverBrush">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutcp">
    <property name="leftMargin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse Select</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutcp_2">
    <property name="leftMargin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBoxSelectBrush">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBorderWith">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
